docs/ios/v2/features/hls-stats.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: HLS Stats (Beta)
+nav: 3.986
+---
+
+If you use Apple's AVPlayer to play HLS stream, you can utilise 100ms SDK to monitor statistics of your HLS playback. You can use these stats to help you debug issues or show useful statistics about for your HLS playback.
+
+## Minimum Requirements
+
+-   SDK version 0.5.1
+-   You must be using AVPlayer for HLS Playback
+
+## How to monitor HLS playback statistics
+
+You create an instance of HMSHLSPlaybackMonitor class passing an instance of AVPlayer that's setup to play an HLS stream.
+
+```swift
+var statMonitor: HMSHLSStatsMonitor?
+
+...
+let player = AVPlayer(playerItem: item)
+statMonitor = HMSHLSStatsMonitor(player: player)
+  ```
+
+That's all. Now you can check the property exposed by HMSHLSStatsMonitor at any moment to get various stats about the current HLS playback.
+  
+HMSHLSStatsMonitor exposes following property to give you stats of current HLS playback:
+
+1. **estimatedBandwidth** - The network bandwidth estimated by the player. This is the bitrate observed by the player in current network conditions (bits per second)
+2. **bitrate** - The bitrate of current HLS stream (layer) indicated by the server (bits per second)
+3. **bytesDownloaded** - Total number of bytes downloaded till this moment (bytes)
+4. **bufferedDuration** - The duration of forward buffer that player has downloaded and ready to play (milliseconds)
+5. **distanceFromLiveEdge** - The distance of current playback position from the live edge of HLS stream (milliseconds)
+6. **droppedFrames** - The total number of frames dropped till this moment
+7. **videoSize** - The resolution of the HLS stream
+8. **watchDuration** - Total duration that the user has watched the stream in milliseconds includes any rewatch time (milliseconds)
+
+## How can I keep monitoring these stats in a time interval
+
+You use a timer to check the stats you are interested in like below:
+
+```swift
+Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) { _ in
+
+  print(statMonitor.estimatedBandwidth)
+  print(statMonitor.distanceFromLiveEdge)
+  
+}
+  ```
+
+## What properties can be observed?
+
+You use HMSHLSPlaybackDelegate on HMSHLSStatsMonitor to listen to some important changes during HLS playback.
```

Comment:
Set the delegate property of HMSHLSStatsMonitor to an instance conforming to  HMSHLSPlaybackDelegate protocol to listen ..